.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,25 @@
+---
+name: "\U0001F680 Feature request"
+about: Suggest a new feature or a big change
+
+---
+<!-- Thank you for contributing. These HTML commments will not render in the issue, but you can delete them once you've read them if you prefer! -->
+
+### Proposed change
+<!-- Use this section to describe the feature you'd like to be added. -->
+
+
+### Alternative options
+<!-- Use this section to describe alternative options and why you've decided on the proposed feature above. -->
+
+
+### Who would use this feature?
+<!-- Describe the audience for this feature. This information will affect who chooses to work on the feature with you. -->
+
+
+### How much effort will adding it take?
+<!-- Try to estimate how much work adding this feature will require. This information will affect who chooses to work on the feature with you. -->
```

Comment:
```suggestion
### Implementation ideas
<!-- Describe what you think needs to be done. -->
```

Reply:
I figure asking for effort required requires some understanding of what needs to be done, asking for a description of that may help us become more actionable and could be the first step towards an indication from a maintainer that a PR doing this would be appreciated.

I'm not confident about my formulation, but I like having a place for the person submitting the feature request to elaborate on what they think may need to be done.

Reply:
I agree that evaluating the effort might be too much in the early stages of an idea. @consideRatio, what do you think about mentioning that offering a solution it's optional and merging the two versions into something like this?
```suggestion
### (Optional): Suggest a solution
<!-- Describe what you think needs to be done. This information will affect who chooses to work on the feature with you -->
```